test/functional/Jsr292/src/com/ibm/j9/jsr292/LookupAPITests_Find.java

```diff
@@ -2106,7 +2107,17 @@ public void test_FindSpecial_Public_CrossPackage_InnerClass() throws Throwable {
 	 */
 	@Test(groups = { "level.extended" })
 	public void test_FindSpecial_Default_CrossPackage_Interface() throws Throwable {
-		MethodHandle example = MethodHandles.lookup().findSpecial(CrossPackageDefaultMethodInterface.class, "addDefault", MethodType.methodType(int.class, int.class, int.class), CrossPackageDefaultMethodInterface.class);
+		try {
+			Class<?> cls = CrossPackageDefaultMethodInterface.class;
+			MethodType mt = MethodType.methodType(int.class, int.class, int.class);
+			MethodHandles.lookup().findSpecial(cls, "addDefault", mt, cls);
```

Comment:
For JDK8, IllegalAccessException is expected so there should be an assert here. Adding a new assert now will cause the current MH impl in OpenJ9 JDK8 to fail. https://github.com/eclipse-openj9/openj9/issues/14987#issuecomment-1795350727 plans to open a new issue to fix the current MH impl. This test should be fixed once OJDK MHs are enabled or the current MH impl is fixed in JDK8.
```suggestion
			MethodHandles.lookup().findSpecial(cls, "addDefault", mt, cls);
			if (VersionCheck.major() <= 8) {
				Assert.fail("[Java 8-] IllegalAccessException not thrown ", e);
```